src/SearchUsersPage/SearchUsersPage.tsx

```diff
@@ -168,7 +171,9 @@ const SearchUsersPage: FC<SearchUsersPageProps> = ({
 							color="secondary"
 							variant="h6"
 						>
-							{result.name}
+							<Link href={`/mastery/${result.name}`} rel="noopener noreferrer">
```

Comment:
We temporarily used the `name` for the link here, but I think the API can now support calls w/ the `summonerId` as we wanted initially. Let's update this `<Link>` and the associated `<Route>` in App to use `summonerId`